src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,6 +302,58 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
     );
   });
+
+  test("Verify that the Premium upsell screen is displayed correctly - overview card", async ({
+    dashboardPage,
+    // purchasePage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463625",
+    });
+
+    //checking that the user can reach upsell page
+    await dashboardPage.goToDashboard();
+    const upsellButton = page.locator("section > div a");
+    await expect(upsellButton).toBeVisible();
+    await upsellButton.click();
+    await page.waitForURL(/.*\/fix\/.*\/view-data-brokers\/?/);
+    const removeThemButton = page.getByText("Remove them for me");
+    await removeThemButton.click();
+
+    //checking the bullet points
+    await page.waitForURL(/.*\/fix\/.*\/automatic-remove\/?/);
+    const ulElement = page.locator("div > strong + ul");
+    expect(await ulElement.isVisible()).toBeTruthy();
```

Comment:
instead of `.isVisible()`+`.toBeTruthy()`, can we use [web first assertions](https://playwright.dev/docs/best-practices#use-web-first-assertions); like `.toBeVisible()` here?